### PR TITLE
Add location addresses data from Ukraine

### PR DESCRIPTION
**Pull request by Pokulita**

### CODE REVIEW

**Review by cieslarmichal — CHANGES_REQUESTED** [review]
Hello, well you need to define actual addresses in location module, modules/location/data and location/Location.cpp and location/LocationTest.cpp like any other countries

**Comment by Pokulita** [comment]
Sure, i will do it. Thanks for the feedback

File: src/modules/location/LocationTest.cpp [hunk 1]
```diff
@@ -26,17 +27,15 @@ using namespace faker;
...
 };
 
 const std::map<AddressCountry, std::string> generatedTestName{
-    {AddressCountry::Usa, "shouldGenerateAmericanAddress"},
-    {AddressCountry::France, "shouldGenerateFrenchAddress"},
-    {AddressCountry::Poland, "shouldGeneratePolishAddress"},
-    {AddressCountry::Russia, "shouldGenerateRussianAddress"},
+    {AddressCountry::Usa, "shouldGenerateAmericanAddress"},    {AddressCountry::France, "shouldGenerateFrenchAddress"},
+    {AddressCountry::Poland, "shouldGeneratePolishAddress"},   {AddressCountry::Russia, "shouldGenerateRussianAddress"},
+    {AddressCountry::Ukraine, "shouldGenerateRussianAddress"},
```
Comment (cieslarmichal) [inline src/modules/location/LocationTest.cpp:38]: it should be "shouldGenerateUkrainianAddress

**Review by cieslarmichal — CHANGES_REQUESTED** [review]

**Comment by cieslarmichal** [comment]
Please fix compilation error and one typo I mentioned in comments.
Also please give a star to the repository to make it more popular :fire:

**Review by cieslarmichal — APPROVED** [review]

File: src/modules/location/LocationTest.cpp [hunk 2]
```diff
@@ -258,6 +257,34 @@ TEST_F(LocationTest, shouldGenerateRussiaStreetAddress)
...
+TEST_F(LocationTest, shouldGenerateUkraineStreetAddress)
+{
+    const auto generatedStreetAddress = Location::streetAddress(AddressCountry::Ukraine);
+
+    ASSERT_TRUE(std::ranges::any_of(ukraineStreetPrefixes, [&generatedStreetAddress](const std::string& prefix)
+                                    { return generatedStreetAddress.find(prefix) != std::string::npos; }));
+    ASSERT_TRUE(std::ranges::any_of(ukraineStreetNames,
+                                    [&generatedStreetAddress](const std::string& streetName)
+                                    {
+                                        return generatedStreetAddress.find(streetName) != std::string::npos;
```
Comment (cieslarmichal) [inline src/modules/location/LocationTest.cpp:283]: it can be either street name or first name or last name :)

Comment (cieslarmichal) [reply src/modules/location/LocationTest.cpp:283]: thus tests are failing

**Review by cieslarmichal — CHANGES_REQUESTED** [review]

**Comment by Pokulita** [comment]
Sorry, I will finish it tomorrow

**Comment by cieslarmichal** [comment]
I will close the PR after 3 days from now if not updated.

**Pull request closed without merging**